### PR TITLE
Clean up and add behave logging

### PR DESCRIPTION
**Pull request by meeuw**
## Description
Clean up and add behave logging

## Checklist

- [x] I've added this contribution to the `changelog.rst`.
- [x] I've added my name to the `AUTHORS` file (or it's already there).

### CODE REVIEW

**Comment by j-bennet** [comment]
👍

File: tests/features/steps/wrappers.py [hunk 1]
```diff
@@ -4,17 +4,44 @@
...
+                {1!r}
+                ---
+                Full log:
+                ---
+                {2!r}
+                ---
+            ''').format(
+                expected,
+                actual,
+                context.logfile.getvalue()
```
Comment (saper) [inline tests/features/steps/wrappers.py]: This seems to cause #997 to me

File: tests/features/steps/wrappers.py [hunk 2]
```diff
@@ -4,17 +4,44 @@
...
+except ImportError:
+    from io import StringIO
 
 
 def expect_exact(context, expected, timeout):
+    timedout = False
     try:
         context.cli.expect_exact(expected, timeout=timeout)
-    except:
+    except pexpect.exceptions.TIMEOUT:
```
Comment (saper) [inline tests/features/steps/wrappers.py]: This seems to cause #996 to me - pexpect 3.0+ does not use `pexpect.exceptions`.

File: tests/features/steps/wrappers.py [hunk 3]
```diff
@@ -4,17 +4,44 @@
 import re
 import pexpect
 from pgcli.main import COLOR_CODE_REGEX
+import textwrap
+
+try:
+    from StringIO import StringIO
+except ImportError:
+    from io import StringIO
```
Comment (saper) [inline tests/features/steps/wrappers.py]: is `StringIO` used anywhere?

**Pull request merged**